src/Apps/Settings/Routes/MyCollection/Components/MyCollectionEmptyState.tsx

```diff
@@ -7,230 +7,150 @@ import {
+                      options: {
+                        title: mode => {
+                          const action = mode === "Login" ? "Log in" : "Sign up"
+                          return `${action} to upload works to My Collection`
+                        },
+                      },
+                      analytics: {
+                        contextModule: ContextModule.myCollectionHome,
+                        intent: Intent.login,
+                      },
+                    },
+                    legacy: {
+                      mode: ModalType.login,
+                      intent: Intent.login,
+                      contextModule: ContextModule.myCollectionHome,
+                      copy: "Log in to upload works to My Collection",
+                    },
+                  })
+
+                  return
+                }
+
+                trackAddCollectedArtwork()
+              }}
+            >
+              Upload Artwork
+            </Button>
+          </Column>
+        </GridColumns>
+
+        {!isLoggedIn && (
+          <>
+            <Spacer y={[2, 4]} />
+
+            <Message variant="info">
               Already have artworks in My Collection?{" "}
               <Clickable
                 textDecoration="underline"
                 onClick={() => {
-                  mediator &&
-                    openAuthModal(mediator, {
+                  showAuthDialog({
+                    current: {
+                      mode: "Login",
+                      options: {
+                        title: mode => {
+                          const action = mode === "Login" ? "Log in" : "Sign up"
+                          return `${action} to view My Collection`
+                        },
+                      },
+                      analytics: {
+                        contextModule: ContextModule.myCollectionHome,
+                        intent: Intent.login,
+                      },
+                    },
+                    legacy: {
                       mode: ModalType.login,
                       intent: Intent.login,
                       contextModule: ContextModule.myCollectionHome,
                       copy: "Log in to view My Collection",
-                    })
+                    },
+                  })
                 }}
               >
                 Log In
               </Clickable>{" "}
               to view them.
-            </Text>
-          </Message>
+            </Message>
+          </>
         )}
       </Column>
 
       <Column span={6}>
-        <ResponsiveBox aspectHeight={652} aspectWidth={770} maxWidth="100%">
+        <ResponsiveBox aspectWidth={910} aspectHeight={770} maxWidth="100%">
           <Image
-            src={image.src}
             width="100%"
             height="100%"
-            srcSet={image.srcSet}
-            lazyLoad
-            alt="Screenshot of My Collection sample screen on the App against a painting background"
```

Comment:
`alt` tags like this aren't actually useful and are against accessibility guidelines. This is a decorative image, essentially.

Reply:
Eh, I suppose this one is debatable. I'll restore this for now; language like "Image of" isn't useful, but "Screenshot of" — I'm not so sure.